### PR TITLE
fix #315

### LINKED ISSUE #315: output author names in editorial
see e.g. A031070 with 11 authors

### PR DESCRIPTION
**Pull request by jakosch**

### CODE REVIEW

**Comment by peterstadler** [comment]
Since this might become a very long list (see A031070) why not make it an inline list and separate by semicolon (via css)? what do you think?
Additionally, the call to `print-forename-surname-from-nameLike-element()` could be replaced with `app:printCorrespondentName()` which would print out a proper link (and not just the name).

**Comment by jakosch** [comment]
Inline List :heavy_check_mark: 

Kannst du mal schauen, wieso app:printCorrespondentName dort offenbar kein node bekommt (oder aus irgendwelchen anderen Gründen kein Link erstellt)?

File: resources/sass/base/_helpers.scss [hunk 1]
```diff
@@ -56,6 +56,16 @@
     margin: 0px 1em;
 }
 
+.inline-list li {
```
Comment (peterstadler) [inline resources/sass/base/_helpers.scss]: maybe add some `padding-right`?

File: templates/document.html [hunk 2]
```diff
@@ -183,6 +183,15 @@ <h3 data-template="lang:translate">series</h3>
                             <p data-template="app:series">WeGA, Briefe, Digitale Edition</p>
                         </div>-->
                         
+                        <div data-template="app-shared:if-exists" data-template-key="authors">
+                            <h3 data-template="lang:translate" class="media-heading">authors</h3>
+                            <ul class="inline-list">
+                                <li data-template="app-shared:each" data-template-from="authors" data-template-to="author">
+                                    <span data-template="app-shared:print" data-template-key="author"/>
```
Comment (peterstadler) [inline templates/document.html]: `app-shared:print()` will only print the atomic value. Use `app-shared:output()` instead to directly output the HTML  ;)

Comment (jakosch) [reply templates/document.html]: :man_facepalming:

File: resources/sass/base/_helpers.scss (hunk 1 above)
Comment (jakosch) [reply resources/sass/base/_helpers.scss]: I think too much padding looks weird, but I'll add a few px for your sake ;)

**Pull request merged**